scripts/autograph_travis_test_config.yaml

```diff
@@ -245,4 +245,7 @@ authorizations:
       key: fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu
       signers:
           - webextensions-rsa
+    - id: bob
+      key: 9vh6bhlc10y63ow2k4zke7k0c3l9hpr8mo96p92jmbfqngs9e7d
+      signers:
```

Comment:
These credentials have to match https://github.com/mozilla-services/autograph/blob/master/autograph.yaml#L1053 (and respectively for `alice`)